javatests/arcs/core/host/HandleAdapterTest.kt

```diff
@@ -209,56 +209,6 @@ class HandleAdapterTest {
         assertThat(writeOnlyHandle).isNotInstanceOf(ReadWriteCollectionHandle::class.java)
     }
 
-    @Test
-    fun singletonHandleAdapter_onUpdateTest() = runTest {
```

Comment:
Why delete these tests?

Reply:
This change breaks them. Because the Handle and its StorageProxy are created in isolation, the proxy will be in the NO_SYNC state and therefore no longer sends the update notification. Other tests cover the event methods more thoroughly, so these aren't all that useful any more.

Reply:
Cool, thanks for the explanation.